renderdoc/replay/replay_driver.h

```diff
@@ -37,6 +37,36 @@ struct FetchFrameRecord
   rdctype::array<FetchDrawcall> drawcallList;
 };
 
+enum RemapTextureEnum
+{
+  REMAP_NONE,
+  REMAP_R8,
+  REMAP_RGBA8,
+  REMAP_RGBA16,
+  REMAP_RGBA32,
+  REMAP_D32S8
+};
```

Comment:
The convention I tend to stick to for enum values in RenderDoc is `eRemap_RGBA8` or `eRemap_RGBA16`. At some point when C++11 features are available for use I'll probably shift to using enum classes and drop the `eEnumType_` prefix in favour of better namespacing.